paymentmethod.go

```diff
@@ -2,6 +2,15 @@ package stripe
 
 import "encoding/json"
 
+// PaymentMethodFPXAccountHolderType is a ist of string values that FPX AccountHolderType accepts.
+type PaymentMethodFPXAccountHolderType string
+
+// List of values that PaymentMethodFPXAccountHolderType can take
+const (
+	AccountHolderTypeIndividual PaymentMethodFPXAccountHolderType = "individual"
+	AccountHolderTypeCompany    PaymentMethodFPXAccountHolderType = "company"
```

Comment:
This needs to be `PaymentMethodFPXAccountHolderTypeIndividual`

Reply:
@remi-stripe  Done. sorry for dragging for so long.